.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,59 @@
-### Prerequisites
+## Prerequisites
 
-- [ ] I have read and understood the [contributing guide][CONTRIBUTING.md].
-- [ ] The commit message follows the [conventional commits][cc] guidelines.
-- [ ] Tests for the changes have been added (for bug fixes / features).
-- [ ] Docs have been added/updated (for bug fixes / features).
+- [ ] I have read and understood the [Contributing Guide][contributing].
+- [ ] My commit message follows the [Conventional Commits][cc] guidelines.
+- [ ] Relevant tests have been added or updated (for bug fixes/features).
+- [ ] Documentation has been added or updated (for bug fixes/features).
```

Comment:
```suggestion
- [ ] Relevant tests have been added or updated (for bug fixes or features).
- [ ] Documentation has been added or updated (for bug fixes or features).
```

Instead of separating phrases with a slash, I think using an "or" is clearer. Just a small suggestion. :)